filebeat/reader/multiline/multiline.go

```diff
@@ -267,6 +268,15 @@ func (mlr *Reader) clear() {
 
 // finalize writes the existing content into the returned message and resets all reader variables.
 func (mlr *Reader) finalize() reader.Message {
+	// add number of lines to event
+	if mlr.numLines > 1 {
+		mlr.message.AddFields(common.MapStr{
+			"log": common.MapStr{
+				"multiline": mlr.numLines,
+			},
+		})
+	}
```

Comment:
Why don't report `log.multiline: 1` ?

Reply:
Can a multiline event contain only 1 line? Reason I'm asking is because I was thinking what if someone wants to see all mutiline events. So far I assumed it would be _exists_:log.multiline.

But your idea below kind of also answers this. If we call it `log.lines` we should probably add it to all events, not only mutiline events?